clients/ctl/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -411,3 +414,54 @@ export const CustomTextArea = ({
     </FormControl>
   );
 };
+
+interface CustomRadioGroupProps {
+  label: string;
+  options: Option[];
+}
+export const CustomRadioGroup = ({
+  label,
+  options,
+  ...props
+}: CustomRadioGroupProps & FieldHookConfig<string>) => {
+  const [field, meta] = useField(props);
+  const isInvalid = !!(meta.touched && meta.error);
```

Comment:
Same here for the double `!!` -- or are we using the double negative for type conversions?

Reply:
yup yup exactly, for the type conversion to boolean to keep typescript happy

Reply:
Gotcha! Thanks for confirming 😄